index.html

```diff
@@ -34,7 +34,7 @@
           <span class="icon-bar"></span>
         </button>
         <a class="navbar-brand" href="/bootstrap-sass">
-          <img src="assets/images/logo.svg" alt="Talis" width="75" height="30">
+          <img src="assets/images/logo.png" alt="Talis" height="40">
```

Comment:
The logo appears not to be centered vertically.

Reply:
I know, I am having issues with that.....